Makefile

```diff
@@ -0,0 +1,19 @@
+clean:
+	echo "Cleaning project"
+	./bootstrap/scripts/cleanup.sh
+
+install: 
+	echo "Installing yarn dependencies..."
+	yarn
+	echo "Installing package dependencies..."
+	yarn setup
+
+watch: 
+	echo "Watching for changes..."
+	./bootstrap/scripts/watch.sh
+
+cleanBuild:
```

Comment:
usually, `building` doesn't imply also `watching`. would drop this last one

Reply:
Done!